### PR TITLE
43 implement a global style sheet

### LINKED ISSUE #43: Implement a global style sheet
## Description
As a Prodigo employee I want uniform styling of the website because I want make a professional impression.

## Acceptance criteria
- [x] There is a style sheet containing the following:
  - [x] Use Prodigo font types
  - [x] Use Prodigo font sizes
  - [x] Use Prodigo colours
- [x] A default style should be applied to everything.
- [x] The style shouldn't be in conflict with Bootstrap-styles

### PR DESCRIPTION
**Pull request by pkasinski**
## Summary
Changes the style sheet to create a uniform styling of the website.

Screen Shot 2022-04-12 at 11 16 52

## Definition of done
- [x] All acceptance criteria are completed.
- [ ] The code has (at least) 80% branch test coverage.
- [ ] All tests pass.
- [ ] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #43

### CODE REVIEW

File: src/Client/wwwroot/css/app.css [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
+}
+
+p {
+    font-size: 17px;
+    line-height: 29px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', Helvetica, Arial, sans-serif;
+}
```
Comment (martinjonsson01) [inline src/Client/wwwroot/css/app.css:28]: It seems like [Prodigo.se](https://prodigo.se/) uses Merriweather only for `h1, h2` and Roboto for `h3, h4, h5, h6`. At least that's what I found, maybe you found something else?

**Review by martinjonsson01 — CHANGES_REQUESTED** [review]
Nice! 

Just a minor detail about the headings.

File: src/Client/wwwroot/css/app.css [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
+    font-size: 55px;
+    line-height: 75px;
+    font-weight: 600;
+    letter-spacing: 0;
+}
+
+h4 {
+    font-size: 26px;
+    line-height: 42px;
+}
```
Comment (martinjonsson01) [inline src/Client/wwwroot/css/app.css:50]: Also, no styling for `h3`?  Maybe they didn't have any of that type of header on their website, but I think it's better if we have the option to use this inbetween-size, so could you maybe give it a size inbetween `h4` and `h2`?

Comment (pkasinski) [reply src/Client/wwwroot/css/app.css:50]: `h3` is not currently used on their website, but I can add it.

**Review by martinjonsson01 — APPROVED** [review]
lgtm 👍

**Review by XwayTomy — APPROVED** [review]
LGTM!

**Review by jonathan-lindqvist — CHANGES_REQUESTED** [review]
Just left one request because it might cause some problems. Otherwise nicely done 👍

File: src/Client/wwwroot/css/app.css [hunk 3]
```diff
@@ -1,21 +1,78 @@
...
+
+h6 {
+    font-size: 18px;
+    line-height: 35px;
+    font-weight: 400;
+    letter-spacing: 0;
+}
```
[the remark this answers is not in the document]
Comment (pkasinski) [reply src/Client/wwwroot/css/app.css:60]: These values are from prodigo.se. Do you think we should use different units?

[the remark this answers is not in the document]
Comment (martinjonsson01) [reply src/Client/wwwroot/css/app.css:60]: We could use `rem` units instead, they're defined (at 100% scaling) as 16px so just divide the current numbers by 16 and replace the unit with `rem`.

**Review by jonathan-lindqvist — APPROVED** [review]
LGTM!

**Pull request merged**